packages/core/src/app/address/googleAutocomplete/AddressSelectorCA.ts

```diff
@@ -0,0 +1,7 @@
+import AddressSelector from './AddressSelector';
+
+export default class AddressSelectorUK extends AddressSelector {
```

Comment:
```suggestion
export default class AddressSelectorCA extends AddressSelector {
```

Reply:
Now you see why I don't like using `default` 🥴

Reply:
Wow that's next level copy paste error 🤦